### ISSUE TITLE
Document process to create blog post and slide presentation for each Kubeflow release

### ISSUE THREAD

**Issue by jbottum** [kubeflow/community#529]
Purpose: The Kubeflow Product Management Team has been asked to document the Kubeflow Community’s process for the blog post and the recorded presentation, which are delivered with each Kubeflow release.   

Goal: The blog post and presentation are used to increase the Community’s understanding of new features in a Kubeflow release.

The current process has been documented in this doc: https://docs.google.com/document/d/1hcd2d-7r8nF1jyFbkXiXA1FugwW_zlgc7OyAKVdWd5c/edit#

Delivery: We will potentially add this as an addendum to the Release Handbook.

Ask: Would members of the Kubeflow Community please review the draft and provide comments by Nov 23?   After those initial comments, we will develop a near-final version, ask for more comments, incorporate input and agree on process to post this process.  Review requested by: @zijianjoy @shannonbradshaw @kimwnasptd @theadactyl

**Comment by zijianjoy** [comment]
That is awesome, thank you Josh for leading this effort! Would you like to share doc access to my account(jamxl@google.com) for review? Thank you!

**Comment by jbottum** [comment]
@zijianjoy you should have access now.

**Comment by theadactyl** [comment]
@jbottum can we make sure this ends up in GH once published? Will TAL by Thursday.

**Comment by zijianjoy** [comment]
Thank you @jbottum, the process documentation is awesome!

**Comment by jbottum** [comment]
@theadactyl yes, but I will need some help.   For instance, it has a table, which might need to be re-formatted once we have agreement on content.   I would like to finish on the content in the next week or so.   I think we have a decent version after incorporating James' input.  I will ask for more comments in the next Community meeting and try to close up by Thanksgiving.

**Comment by zijianjoy** [comment]
@jbottum If you want to convert table to markdown, you can use tools like https://tabletomarkdown.com/convert-spreadsheet-to-markdown/ for your convenience.

**Comment by thesuperzapper** [comment]
I think this should be included under the `releases` folder created by; https://github.com/kubeflow/community/issues/531

**Comment by theadactyl** [comment]
@thesuperzapper agreed. Otherwise LGTM

**Comment by kimwnasptd** [comment]
> I think this should be included under the releases folder created by; #531

Yep, we discussed about it in the release team meeting this Monday. We will move these here once we move our release docs over this repo cc @annajung

**Comment by zijianjoy** [comment]
Hello @jbottum , would you like to create a PR for this process in Markdown format? Thank you again for your work on documenting the media workflow!

**Comment by jbottum** [comment]
@zijianjoy this is on my to-do list and will get it going, thanks for the reminder.

**Comment by zijianjoy** [comment]
Thank you so much @jbottum , I am following up on this topic and see if there is any update?

**Comment by jbottum** [comment]
I copied the doc, then I edited the chart in the new doc and converted it to markdown (using google doc add-on). 
 The work is here: https://docs.google.com/document/d/18jIr1oy_hW0xC9JDR4stA56jAN_WsgaJHkeIhhN9pLI/edit?usp=sharing

I have asked @shannonbradshaw to help me with the PR creation and provided him with the markdown.   I expect a last round of edits once the PR is created and formatting looks okay.

**Comment by zijianjoy** [comment]
That is awesome! Thank you Josh and looking forward to the next step.

**Comment by jbottum** [comment]
@shannonbradshaw let's post this on this page: https://github.com/kubeflow/community/tree/master/releases

I think it could fit under resources :

Resources

Release Handbook
Meeting Recordings
Please add ->  Process to publish a Blog Post and Video Presentation for a Kubeflow Release

**Comment by juliusvonkohout** [comment]
/close

Due to inactivity for two years
